cactus/tests/test_basic.py

```diff
@@ -113,4 +114,30 @@ def test_current_page(self):
             self.assertEqual('True', f.read())
 
         with open(os.path.join(self.path, '.build', other), 'rU') as f:
-            self.assertEqual('False', f.read())
\ No newline at end of file
+            self.assertEqual('False', f.read())
+
+
+class TestPage(SiteTestCase):
```

Comment:
It doesn't *need* to be a separate class. But I wanted to separate the more granular tests for `Page` from the tests for `Site`.